### ISSUE TITLE
Bug with v-model.trim

### ISSUE THREAD

**Issue by hwatson381** [eee555/saolei_website#45]
There's a bug with using v-model.trim on el-input.

This is a known issue with element-ui and it prevents the user from typing spaces (for example, in the real name field).
https://github.com/ElemeFE/element/issues/19165

There is a slightly separate issue, which is that the real name input box on the profile page has max-length="10". This is too short for English names. I think setting the limit to 50 characters would work for almost all names.

**Comment by eee555** [comment]
Yeah, you're right. We will take into account input experiences in different languages.
